docs/deployment/k8s/index.mdx

```diff
@@ -212,34 +163,36 @@ spec:
             source_config:
               - name: default-http
                 properties:
-                  ADDRESS: 127.0.0.1:8080
-
+                  address: 127.0.0.1:8080

```

Comment:
Running this on `127.0.0.1` will be problematic inside Kubernetes, let's use `0.0.0.0` instead:
```suggestion
                  address: 0.0.0.0:8080
```